static/app/components/organizations/pageFilters/parse.tsx

```diff
@@ -1,8 +1,11 @@
+import {Location} from 'history';
```

Comment:
this file contains `parses` and `normalizations` . should we rename it so that it covers both types?

Reply:
what might you propose?

Reply:
hm atm I don't have a good suggestion 😄